### PR TITLE
update sponsorship information and kill form

### PR DESCRIPTION
**Pull request by danabauer**
@kenperkins  please review the new sponsorship text and then tag me so I can do a final copy edit before merging. I plan to add more specific information about contributions, sponsorships, and OSS hosting participants in the next round of updates.

### CODE REVIEW

File: src/site_source/community/index.html [hunk 1]
```diff
@@ -7,155 +7,51 @@
...
+  <div class="community container body-container">
+    <div class="row">
+      
 
-      <div class="sponsorship-result alert hidden"></div>
-      <div class="sponsorship-form">
-        <p>Please fill out and submit this form. We're already excited about your event!</p>
+      <div class="content col-md-9">
+        <p class="lead">
+        Rackspace is committed to supporting developers and technologists. Our developer advocates write code and documentation and build tools to make the Rackspace Cloud Platform and OpenStack easy to use. We give talks, organize conferences, teach workshops, and field questions from people around the world. We also offer sponsorships for technical events and discounted hosting for OSS projects. Let us know what kind of support you need, and a Racker will reach out to you.
```
Comment (rgbkrk) [inline src/site_source/community/index.html]: "...developer advocates write code, document, and build tools..."

File: src/site_source/community/index.html [hunk 2]
```diff
@@ -7,155 +7,51 @@
...
-        </form>
-      </div>
-    </div>
-    <div class="sidebar faq col-md-3">
-      <p class="sidebarheader">What type of events does Rackspace sponsor?</p>
-      <p>We sponsor large and small conferences, meetups, workshops, hackathons, and other kinds of tech community gatherings.</p>
+      
+      
+        <h2>Open Source Contributions</h2>
+        <p>Open source contributions are part of the job description for many Rackers. We build tools in-house and share with the community; we accept pull requests and we mentor and answer questions; and we design and implement new tools in the open. We also contribute to open source projects outside of the Rackspace and OpenStack ecosystems.
```
Comment (rgbkrk) [inline src/site_source/community/index.html]: I'd break up sections like 

> We build tools in-house and share with the community; we accept pull requests and we mentor and answer questions; and we design and implement new tools in the open.

into a series of bullets. The semi-colons + ands make it sound very run on. I need to catch my breath here.

File: src/site_source/community/index.html [hunk 3]
```diff
@@ -7,155 +7,51 @@
...
+      
+        <h2>Open Source Contributions</h2>
+        <p>Open source contributions are part of the job description for many Rackers. We build tools in-house and share with the community; we accept pull requests and we mentor and answer questions; and we design and implement new tools in the open. We also contribute to open source projects outside of the Rackspace and OpenStack ecosystems.    
+        </p>
+
+      
+      
+      
+        <h2>Event Sponsorship Program</h2>
+        <p>In 2014, Rackspace developers and developer advocates attended, sponsored, spoke at, and facilitated hundreds of conferences, meetups, and workshops for developers and technologists around the world. This "fanatical support" in the field is central to Rackspace's mission: we want to meet developers where they are, in their communities, and show them the love and support they need to build amazing things on the Rackspace Cloud Platform. For more information about our Events Sponsorship Program, contact us at <a href="mailto:sponsorships@rackspace.com?Subject=Event%20Sponsorship" target="_top">sponsorships@rackspace.com</a>.</p>
```
Comment (rgbkrk) [inline src/site_source/community/index.html]: :+1:

File: src/site_source/community/index.html [hunk 4]
```diff
@@ -7,155 +7,51 @@
...
 
-      <p class="sidebarheader">What kind of sponsorships do you provide?</p>
+        <h2>OSS Hosting Program</h2>
+        <p>Since 2013, Rackspace has provided generous discounts on hosting for more than 150 OSS projects and communities. We support a wide range of projects from the security, scientific computing, big data, open hardware, and Python communities, among others. For more information about our OSS Hosting Program, contact us at <a href="mailto:sponsorships@rackspace.com?Subject=OSS%20Hosting" target="_top">sponsorships@rackspace.com</a>.</p>
```
Comment (rgbkrk) [inline src/site_source/community/index.html]: Can't wait until we can have a featurette/gallery of the projects and communities.

**Comment by danabauer** [comment]
Thanks @rgbkrk. I just pushed some changes. Waiting for @kenperkins to review.

**Comment by kenperkins** [comment]
reviewing now

**Comment by kenperkins** [comment]
And of course my dev environment is failing to build.

I'll do a code-review instead.

File: src/site_source/community/index.html [hunk 5]
```diff
@@ -7,155 +7,62 @@
...
+      </div>
+      
+      <div class="sidebar faq col-md-3">
+        <p class="sidebarheader">What type of events does Rackspace sponsor?</p>
+        <p>We sponsor large and small conferences, meetups, workshops, hackathons, and other kinds of tech community gatherings. We like to be hands-on with the communities we support, so we almost always attend the events we sponsor.</p> 
 
+        <p class="sidebarheader">What kind of sponsorships do you provide?</p>
 
-      <p class="sidebarheader">Can I get a Racker to speak or teach at my event?</p>
+        <p>Usually we offer financial support through our Events Sponsorship Program, but we can also provide space for meetups and workshops in our Austin, San Francisco, and Blacksburg offices. Let us know what you're looking for. We also offer discounted hosting for open source projects and open communities through our OSS Hosting Program.</p>
```
Comment (kenperkins) [inline src/site_source/community/index.html]: I feel like the offer of space for hosting events could get us in hot water with those offices. Do we have the latitude to make this claim?

File: src/site_source/community/index.html [hunk 6]
```diff
@@ -7,155 +7,62 @@
...
+
+      
+      
+      
+        <h2>Event Sponsorship Program</h2>
+        <p>Rackspace developers and developer advocates attend, sponsor, speak at, and facilitate hundreds of events around the world. This "fanatical support" in the field is central to Rackspace's mission: we want to meet developers where they are, in their communities, and show them the love and support they need to build amazing things on the Rackspace Cloud Platform. For more information about our Events Sponsorship Program, contact us at <a href="mailto:sponsorships@rackspace.com?Subject=Event%20Sponsorship" target="_top">sponsorships@rackspace.com</a>.</p>
 
-      <p class="sidebarheader">What kind of sponsorships do you provide?</p>
+        <h2>OSS Hosting Program</h2>
+        <p>Rackspace provides generous discounts on hosting for more than 150 OSS projects and communities. We support a wide range of projects in the security, scientific computing, big data, open hardware, and Python communities, among others. For more information about our OSS Hosting Program, contact us at <a href="mailto:sponsorships@rackspace.com?Subject=OSS%20Hosting" target="_top">sponsorships@rackspace.com</a>.</p>
```
Comment (kenperkins) [inline src/site_source/community/index.html]: Do you think we can keep the section mentioning the OSS program, but without the contact info?

**Comment by danabauer** [comment]
@kenperkins the Blacksburg and Austin offices and Geekdom in SF regularly host meetups. But I can remove the specific office locations if you'd like. Yes, I can remove the contact information from the OSS program section.

**Comment by danabauer** [comment]
I updated the option in Community drop-down menu (in flighted-header.html) to Outreach & Events.

**Comment by ChristieQ** [comment]
@danabauer - Looks like the global nav is still serving from VWO, so I don't think your changes are visible across most browsers. @kenperkins is in the process of moving the new nav code into production, but I'm not sure where that currently stands.

**Comment by danabauer** [comment]
Do we need to hold off on this? Or can we kill the form and update the text? The Community nav might be a little strange (until the new nav code gets pushed to production) but it will still be functional.

**Comment by kenperkins** [comment]
We should be able to edit in VWO.

**Comment by kenperkins** [comment]
@danabauer we should probably open an issue to decommission the sponsorship form webserver.

**Comment by danabauer** [comment]
new-nav

Ken temporarily disabled the VWO campaigns, and it looks like the new nav with Outreach & Events was implemented. Can we go ahead and merge this PR?

**Comment by kenperkins** [comment]
Yep. You can self merge or I can when I get downstairs

Sent from my iPhone

On Feb 23, 2015, at 7:18 AM, Dana Bauer <notifications@github.com<mailto:notifications@github.com>> wrote:


[new-nav]<https://cloud.githubusercontent.com/assets/751594/6330305/0377a1a2-bb45-11e4-8c7e-553034eefc16.png>

Ken temporarily disabled the VWO campaigns, and it looks like the new nav with Outreach & Events was implemented. Can we go ahead and merge this PR?

**Comment by kenperkins** [comment]
w00t. https://developer.rackspace.com/community/

**Pull request merged**